src/main/webapp/script.js

```diff
@@ -14,11 +14,17 @@
 
 // Fetches information returned from Spoonacular (after the image has been classified appropriately)
 function getRecipeInfo() {
-  const request = new Request('/dishAnalysis', {method: "POST"});
+  const image = document.getElementById('image').files[0];
+  const params = new FormData();
+  params.append('image', image);
+  const request = new Request('/dishAnalysis', {method: "POST", body: params});
   fetch(request).then(response => response.json()).then((recipeListInfoJson) => {
+    var recipeList = JSON.parse(JSON.parse(recipeListInfoJson));
     const displayRecipeElement = document.getElementById('display-recipes');
     displayRecipeElement.innerHTML = "";
-    recipeListInfoJson.forEach(recipe => displayRecipeElement.appendChild(createRecipeElement(recipe)));
```

Comment:
Just be curious, what's the issue with forEach?

Reply:
recipe wasn't giving us back the json that we needed to input into createRecipeElement() when we were using forEach.